File: app/src/main/java/com/app/rupiksha/models/PanDetailsModel.java
package com.app.rupiksha.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PanDetailsModel {
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("pan_status")
    @Expose
    private String panStatus;
    @SerializedName("gender")
    @Expose
    private String gender;
    @SerializedName("dob")
    @Expose
    private String dob;
    @SerializedName("constitution")
    @Expose
    private String constitution;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPanStatus() {
        return panStatus;
    }

    public void setPanStatus(String panStatus) {
        this.panStatus = panStatus;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public String getDob() {
        return dob;
    }

    public void setDob(String dob) {
        this.dob = dob;
    }

    public String getConstitution() {
        return constitution;
    }

    public void setConstitution(String constitution) {
        this.constitution = constitution;
    }
}

File: app/src/main/java/com/app/rupiksha/models/PanDetailsModel.kt
package com.app.rupiksha.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class PanDetailsModel(
    @SerializedName("name")
    @Expose
    val name: String? = null,
    
    @SerializedName("pan_status")
    @Expose
    val panStatus: String? = null,
    
    @SerializedName("gender")
    @Expose
    val gender: String? = null,
    
    @SerializedName("dob")
    @Expose
    val dob: String? = null,
    
    @SerializedName("constitution")
    @Expose
    val constitution: String? = null
)
